recipe/recipe.yaml

```diff
@@ -0,0 +1,77 @@
+context:
+  name: rustworkx
+  python_name: rustworkx
+  version: "0.16.0"
+  python_min: "3.9"
+
+package:
+  name: rustworkx
+  version: ${{ version }}
+
+source:
+  url: https://github.com/Qiskit/${{ name }}/archive/refs/tags/${{ version }}.tar.gz
+  sha256: ef3c6fb3bd7e6a18919023330f3961702346d8afbe95aa5f446f9b222ee72391
+
+build:
+  number: 2
+  script:
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+    - ${{ PYTHON }} -m pip install . -vv
+  skip: not (match(python, python_min ~ ".*") and is_abi3)
+  python:
+    version_independent: true
+
+requirements:
+  build:
+    - if: build_platform != target_platform
+      then:
+      - python
+      - cross-python_${{ target_platform }}
+    - ${{ compiler('c') }}
+    - ${{ stdlib('c') }}
+    - ${{ compiler('rust') }}
+    - cargo-bundle-licenses
```

Comment:
I wasn't sure if `compiler('cxx')` and `make` were still needed so I tried dropping them to see. We can put them back if the build fails. It worked for me locally but I didn't try to isolate my build environment from the system make and c++ compiler. Also, I didn't try to test cross-compiling locally.

Reply:
The build ran okay in CI as well.

Reply:
I think we only need C. When I tried to release for musl in the main repo, Alpine Linux kind of blew up without GCC.

C++ should be safe to remove

Reply:
Resolving